src/sentry/apidocs/examples/project_examples.py

```diff
@@ -378,6 +378,83 @@
     "highlightPreset": {"tags": [], "context": {}},
 }
 
+PROJECT_SUMMARY = {
+    "team": {
```

Comment:
can we reuse the properties of BASE_PROJECT here?

Reply:
trying to reuse what already existed. no idea what base_project and this one have in common